terraform/modules/keycloak/output.tf

```diff
@@ -0,0 +1,7 @@
+output "instance_id" {
+  value = aws_instance.nat_instance.id
```

Comment:
Why the ID is ID of NAT instance?

Reply:
đã thay đổi để lấy được id của keycloak_instance và cập nhật vào file output.tf